### PR TITLE
Deploy the site

### PR DESCRIPTION
**Pull request by anthonylavado**
Merging dev to master automatically deploys on GitHub Pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Jellyfin.media
+
+The official Jellyfin website.
+
+Uses the theme ["Evie" theme](https://github.com/anges244/evie), copyright (c) 2018 [Aggelos Gesoulis](https://twitter.com/anges244) - [Katerina Limpitsouni](https://twitter.com/ninalimpi), available under the MIT license.
+
+The header photo is by [freestocks.org](https://unsplash.com/photos/11SgH7U6TmI?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText), with minor edits.
```
Comment (JustAMan) [inline README.md]: shouldn't we clean these `utm_source=` things?

File: dashboard.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (JustAMan) [inline dashboard.html]: what is this file used for?

File: page.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```
Comment (JustAMan) [inline page.html]: and why we need this page?

File: README.md (hunk 1 above)
Comment (anthonylavado) [reply README.md]: Hrm. Yeah, they’re link source tracking. I just took the attribution link as they gave it to me. Will do.

File: dashboard.html (hunk 2 above)
Comment (anthonylavado) [reply dashboard.html]: Nothing yet! It’s in the template but unused at the moment. Forgot I copied that in there... do plan on potentially using it or something like it in the future.

File: page.html (hunk 3 above)
Comment (anthonylavado) [reply page.html]: Same as above.

**Pull request merged**